backend/accessibility_templates/aa-003.xml

```diff
@@ -0,0 +1,16 @@
+<AccessibilityAttributes>
+ <SubtitleAttributes>
+  <!-- WVTT -> contentType="text" codecs="wvtt" lang="en" -->
+  <Carriage href="urn:tva:metadata:cs:SubtitleCarriageCS:2023:99"/>  <!--other-->
+  <Coding href="urn:tva:metadata:cs:SubtitleCodingFormatCS:2023:4"/>  <!--WebVTT-->
+  <SubtitleLanguage>fra</SubtitleLanguage>
+  <SuitableForTTS>false</SuitableForTTS>
+ </SubtitleAttributes>
+ <AudioDescriptionAttributes>
+  <AudioAttributes>
+   <AudioLanguage>esp</AudioLanguage>
+   <BitRate>2422</BitRate>
```

Comment:
I am not sure if the BitRate property needs to be specified here. In A177r6 examples this property is missing.

Reply:
Indeed, `<Bitrate>` is not included in the A177 profile of `<AudioAttributes>` - see clause 6.10.10 of A177r6.
Will be removed from the accessibility templates.